countVowels-findAverage.js

```diff
@@ -0,0 +1,65 @@
+// Exercise 13
+
+/*
+ Exercise 13 number 1:
+ Write a function called countVowels that takes a string as an argument
+ and returns the number of vowels (a, e, i, o, u) in that string. 
+ For example, If the input is "hello", the function should return 2.
+*/
+
+ // -Define strings of sentence
+ // -Initialize a variable to hold the count
+ // -Loop through with for of Loop
+ // -Initialize another variable to hold the transformation of string to lower case
+ // -Am IF statement to compare vowels with the second variable and add it to count
+ // -Finally return count
+
+ const aboutMe = "I am a Developer";
```

Comment:
Kindly change this variable name to match the one passed to the function (sentence).